### PR TITLE
[CFM-642-dix] Adding photo to photo album.

### PR DESCRIPTION
**Pull request by anvmn**
https://issuetracker.amplexor.com/jira/browse/CFM-642

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/content/c4m_content_photoalbum/c4m_content_photoalbum.module [hunk 1]
```diff
@@ -140,14 +140,22 @@ function c4m_content_photoalbum_form_node_gallery_api_plupload_form_alter(&$form
...
-  if (strpos($form['upload']['#plupload_settings']['url'], '/node-gallery/json/item/create') !== 0) {
+  $position = strpos($upload_url, '/node-gallery/json/item/create');
+  if ($position === FALSE) {
     return;
   }
 
+  // In case site is hosted on custom path-based domain (www.example.com/foo),
+  // cut the path which comes after domain. It will be added by url().
+  if ($position != 0) {
+    $upload_url = substr( $upload_url, $position);
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/content/c4m_content_photoalbum/c4m_content_photoalbum.module]: remove extra space

**Comment by ordavidil** [comment]
@anvmn 
please add a gif.

**Comment by anvmn** [comment]
anim

**Comment by anvmn** [comment]
@ordavidil 
GIF added...

**Comment by ordavidil** [comment]
@anvmn 
Can you please add a gif to show it's working for the regular domain as well ?
thanks

**Comment by anvmn** [comment]
Regular domain:
anim

**Comment by anvmn** [comment]
@ordavidil 
Added

**Pull request merged**